distributed/actor.py

```diff
@@ -168,7 +168,12 @@ async def run_actor_function_on_worker():
                             raise OSError("Unable to contact Actor's worker")
                     return result
 
-                q = asyncio.Queue(loop=self._io_loop.asyncio_loop)
+                loop = asyncio.get_event_loop()
```

Comment:
How does this construct compares with the problem perceived by #5353?

Reply:
That seems to have been a better fix.